assets/js/trainingsplan.js

```diff
@@ -421,7 +439,7 @@ function calculateTime() {
     moduleList = moduleList.filter(el => el.nodeName.includes('LI'));
     for (mod of moduleList) {
         if (mod.className.includes(CLASS_MODULE)) {
-            const duration = parseInt(mod.dataset.duration);
+            const duration = 0;
```

Comment:
The module duration is now represented by the Introduction `Time Break` so it's added automatically to the total time when `calculateTime()` is called. I had to remove it from `const duration = parseInt(mod.dataset.duration);` to avoid duplication